zap/src/main/java/org/parosproxy/paros/extension/option/OptionsDatabasePanel.java

```diff
@@ -172,7 +196,15 @@ public OptionsDatabasePanel() {
                 LayoutHelper.getGBC(0, 5, 1, 0.5D, new Insets(2, 2, 2, 2)));
         panel.add(
                 comboNewSessionOption(),
-                LayoutHelper.getGBC(1, 5, 1, 1.0D, new Insets(2, 2, 2, 2)));
```

Comment:
This is confusing if there’s no 1,5,1 why jump to 1,6,1?

Reply:
I think I fat fingered something along with learning as I go and didnt realise I had done this. With formatting fixed it has made reviewing myself alot easier, thank you so much for having a stab at my low quality PR and for the guidance :)